plugins/org.jkiss.dbeaver.core/src/org/jkiss/dbeaver/ui/perspective/DBeaverPerspective.java

```diff
@@ -40,6 +40,11 @@ public class DBeaverPerspective implements IPerspectiveFactory
     public static final String BOTTOM_BOTTOM_LEFT = "bottomLeft"; //$NON-NLS-1$
     public static final String FOLDER_BOTTOM_RIGHT = "bottomRight"; //$NON-NLS-1$
 
+    // Every time the perspective changes, the version must be bumped.
+    // This also includes perspective extensions that contribute views.
+    // The version is not incremental.
+    public static final String PERSPECTIVE_VERSION = "25.1.2"; //$NON-NLS-1$
```

Comment:
25.1.4?

Reply:
No. This would result in another wipe. Please see the comment; we didn't change the perspective